Sources/FluentKit/Properties/Property.swift

```diff
@@ -78,18 +78,27 @@ extension AnyModel {
     }
     
     var properties: [(String, AnyProperty)] {
-        return Mirror(reflecting: self)
-            .children
-            .compactMap { child in
-                guard let label = child.label else {
-                    return nil
-                }
-                guard let property = child.value as? AnyProperty else {
-                    return nil
-                }
-                // remove underscore
-                return (String(label.dropFirst()), property)
+        var props: [(String, AnyProperty)] = []
+        var mirror: Mirror = Mirror(reflecting: self)
+        while true {
+            props.append(contentsOf: mirror.children
+                .compactMap { child in
+                    guard let label = child.label else {
+                        return nil
+                    }
+                    guard let property = child.value as? AnyProperty else {
+                        return nil
+                    }
+                    // remove underscore
+                    return (String(label.dropFirst()), property)
+            })
+            if let m = mirror.superclassMirror {
```

Comment:
Could this be written as:
```swift
var currentMirror: Mirror? = Mirror(reflecting: self)
while let mirror = currentMirror {
    // add props
    currentMirror = mirror.superclassMirror
}
```

Reply:
Do you like this any better?

```swift
var mirror: Mirror = Mirror(reflecting: self)
while true {
  // add props
  guard let nextMirror = mirror.superclassMirror else { return props }
  mirror = nextMirror
}
```

Reply:
That's a little better. `while true` is a little confusing to me though. When I see that I think this loop should be going forever. Putting the stop condition in the `while` makes it clearer what the loop is doing.